Bottom: 369e2294c0801ce6163a5a874fb6f944c32ae391
Top:    33da41fc5dc0b0329a542d61940533c72e94a41c
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-26 20:17:09 -0400

Refresh of efika-audio

---

diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index 63ab663..5dbebf8 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -56,11 +56,3 @@ config SND_MPC52xx_SOC_EFIKA
 	help
 	  Say Y if you want to add support for sound on the Efika.
 
-config SND_MPC52xx_SOC_EFIKA
-	tristate "SoC AC97 Audio support for bbplan Efika and STAC9766"
-	depends on PPC_EFIKA
-	select SND_SOC_MPC5200_AC97
-	select SND_SOC_STAC9766
-	help
-	  Say Y if you want to add support for sound on the Efika.
-
